templates/angular.webapp/.gitignore

```diff
@@ -0,0 +1,46 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
+
+# compiled output
+/build
+/tmp
+/out-tsc
+# Only exists if Bazel was run
+/bazel-out
+
+# dependencies
+/node_modules
+
+# profiling files
+chrome-profiler-events*.json
+speed-measure-plugin*.json
+
+# IDEs and editors
+/.idea
+.project
+.classpath
+.c9/
+*.launch
+.settings/
+*.sublime-workspace
+
+# IDE - VSCode
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+.history/*
+
+# misc
+/.sass-cache
+/connect.lock
+/coverage
+/libpeerconnection.log
+npm-debug.log
+yarn-error.log
+testem.log
+/typings
+
+# System Files
+.DS_Store
+Thumbs.db
```

Comment:
There is a problem with this idea, because then this one .gitignore file must handle all possible frontend frameworks and could get quite complex, which would be unnecessary for the generated projects as those would only need the angular OR react OR vue .gitignore file . I think there is nothing wrong with nested .gitignore files in this case

Reply:
I also prefer having a single `.gitignore` per repository, but as the current Goxygen implementation doesn't support (yet) any way to separate framework-specific files we can keep it like this.